controller/api/destination/watcher/endpoints_watcher.go

```diff
@@ -49,6 +51,7 @@ type (
 		OwnerKind         string
 		Identity          string
 		AuthorityOverride string
+		Topology          map[string]string
```

Comment:
Topology is a slightly weird name for this field since these are more like topology coordinates.  Maybe "TopologyLabels"?

Reply:
Agreed, will switch it around.